requirements-transitive.txt

```diff
@@ -17,14 +17,14 @@ certifi==2020.12.5
 cffi==1.14.5
 chardet==4.0.0
 click==7.1.2
-confluent-kafka==1.6.0
+confluent-kafka==1.9.2
```

Comment:
Are there any notable changes here? This is a pretty bug version jump.

Reply:
Nothing seems particularly notable https://github.com/confluentinc/confluent-kafka-python/releases

I just did a version bump because I was unable to build locally. I see this in the 1.9.2 release notes `Wheel for macOS M1/arm64`, which would explain that.